src/modules/filelist/File.jsx

```diff
@@ -47,7 +47,7 @@ const File = ({
   onFolderOpen,
   onFileOpen,
   disableSelection = false,
-  isExternalDrive
+  externalDrive
```

Comment:
Usual I prefer to prefix with an `is` the boolean but the props just below don't follow this convention so its only a nit